src/components/KModalFullscreen/KModalFullscreen.vue

```diff
@@ -302,42 +303,43 @@ $fullscreen-modal-padding: 64px;
 .k-modal-fullscreen-title {
   display: inline-flex;
   justify-content: center;
-  margin-bottom: var(--spacing-xs, spacing(xs));
+  margin-bottom: var(--spacing-xs, var(--kui-space-40, $kui-space-40));
   position: relative;
 
-  @media (min-width: $viewport-md) {
+  @media (min-width: $kui-breakpoint-phablet) {
     justify-content: flex-start;
-    margin-bottom: 0;
-    margin-left: 36px;
+    margin-bottom: var(--kui-space-0, $kui-space-0);
+    margin-left: var(--kui-space-90, $kui-space-90); // This was 36px, switched it to 32px
   }
 }
 
 .k-modal-fullscreen-action {
   display: inline-flex;
   justify-content: center;
-  margin-right: var(--spacing-xl, spacing(xl));
+  margin-left: var(--kui-space-50, $kui-space-50) !important;
+  margin-right: var(--spacing-xl, var(--kui-space-90, $kui-space-90));
 
   & button,
   & :deep(button) {
-    font-size: 13px;
-    font-weight: 600;
-    height: 40px;
-    line-height: 13px;
-    margin-left: var(--spacing-md, spacing(md));
+    font-size: var(--kui-font-size-20, $kui-font-size-20);
+    font-weight: var(--kui-font-weight-semibold, $kui-font-weight-semibold);
+    height: var(--kui-space-100, $kui-space-100);
+    line-height: var(--kui-line-height-10, $kui-line-height-10);
```

Comment:
The line height token should typically match the font size token
```suggestion
    line-height: var(--kui-line-height-20, $kui-line-height-20);
```